tests/test_profile/test_quota/test_model.py

```diff
@@ -0,0 +1,49 @@
+from profiles.models import BillingGroup
+from service_catalog.models import Instance
+from tests.test_profile.test_quota.base_test_quota import BaseTestQuota
+
+
+class TestQuotaModel(BaseTestQuota):
+
+    def setUp(self):
+        super(TestQuotaModel, self).setUp()
+
+    def test_consumed_updated_after_billing_group_changed_in_instance(self):
```

Comment:
check resource consumption data. We remove 30 vcpu fron one side and we retrieve them on the other side